python/lsst/ip/diffim/imageDecorrelation.py

```diff
@@ -28,19 +28,21 @@
 import lsst.afw.image as afwImage
 import lsst.meas.algorithms as measAlg
 import lsst.afw.math as afwMath
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
 
-__all__ = ("decorrelateExposure", "computeDecorrelationKernel", "doConvolve",
+__all__ = ("ImageDecorrelationTask", "ImageDecorrelationConfig",
            "computeClippedImageStats")
 
 
-def computeClippedImageStats(im):
+def computeClippedImageStats(im, numSigmaClip=3., numIter=3):
```

Comment:
I suggest removing this function entirely. You only call it in two places in this file, and those two places can use the same statsControl object, so it's very few extra lines of code. Furthermore, in this file you ignore two out of the three statistics that you compute (you apparently only use the variance). I don't feel this is a good function to make public, either.